README.md

```diff
@@ -21,15 +21,16 @@ gem "merch_calendar"
 
 ## Usage
 
-For converting a date into a `MerchWeek` object.
+For converting a date into a `MerchWeek` object.  `MerchWeek` object can take a date can translate to what merch week the date falls into. 
+
 
 ```ruby
-merch_week = MerchCalendar::MerchWeek.from_date("2014-01-01")
+merch_week = MerchCalendar::MerchWeek.from_date("2014-01-01", calendar: MerchCalendar::RetailCalendar.new)
 
 merch_week.year                 # 2013 (the merch year associated with this date)
 merch_week.month                # 12 (the julian month that the date falls in)
 merch_week.week                 # 5 (the week number within the month)
-merch_week.year_week            # 48 (the week number within the year)
+merch_week.year_week            # 48 (the week number within the retail calendar year)
 merch_week.quarter              # 2
```

Comment:
Shouldn't this be `4`, since we're talking about a week in January?

Reply:
yep! good catch! 👍